pkg/asset/internal/templates.go

```diff
@@ -419,4 +420,42 @@ spec:
     port: 53
     protocol: TCP
 `)
+	EtcdOperatorTemplate = []byte(`apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: etcd-operator
+  namespace: kube-system
+  labels:
+    k8s-app: etcd-operator
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: etcd-operator
+    spec:
+      containers:
+      - name: etcd-operator
+        image: quay.io/coreos/etcd-operator
```

Comment:
@aaronlevy 
We have only one tag version "v0.1.0", which isn't useful for this.
Mind wait until "v0.2.0"?